app/components/question.tsx

```diff
@@ -80,12 +88,18 @@ export default function Question({
                 ref={answerRef}
               />
               <div className="actions">
+                <ThumbsUp />
```

Comment:
ah, we shouldn't show the thumbs up button until #19 is implemented to store the upvote, and that one is probably dependent on adding that feature to stampy's Flask server without directly calling the pageforms autoedit MediaWiki API (which has too much power to open up for anonymous use) => it was moved to "gamma" milestone (~August?)

@plexish do you know what other action icons are ready besides the Edit button?

Reply:
Should I remove everything except for the Edit button for now?

Reply:
yeah, I would remove everything except Edit for now